command/format/diagnostic.go

```diff
@@ -213,6 +213,10 @@ func DiagnosticWarningsCompact(diags tfdiags.Diagnostics, color *colorstring.Col
 }
 
 func appendSourceSnippets(buf *bytes.Buffer, diag *viewsjson.Diagnostic, color *colorstring.Colorize) {
+	if diag.Address != "" {
+		fmt.Fprintf(buf, "  (%s)\n", diag.Address)
```

Comment:
Would formatting this in a sentence-fragment-like form make sense? Sort of in line with the summary immediately below it. For example:

```
│ Error: something went horribly wrong
│
│   in module.mod["foo"].aws_vpc.a["bar"]
│   on mod/main.tf line 2, in resource "aws_vpc" "a":
```

Reply:
That was exactly the first way I formatted the error, but the 3 fragments together felt a little strange. I think mostly it was the _"in ..., on ..., in ..."_ didn't seem quite right.

I'm not tied to any particular format here, so if anyone wasn't to see it decorated in another way, we can change it up.

Reply:
OK, last bit of bike-shedding, since the error is "in" a configuration body, I'm trying "with" for the instance address.

Reply:
I am all in favour of increasing the number of prepositions in the UI! Reads nicely to me 👍